src/logic/currencyValues/store/actions/fetchCurrencyRate.ts

```diff
@@ -1,13 +1,18 @@
+import { Action } from 'redux-actions'
+import { ThunkDispatch } from 'redux-thunk'
+
 import fetchCurrenciesRates from 'src/logic/currencyValues/api/fetchCurrenciesRates'
 import { setCurrencyRate } from 'src/logic/currencyValues/store/actions/setCurrencyRate'
 import { AVAILABLE_CURRENCIES } from 'src/logic/currencyValues/store/model/currencyValues'
-import { Dispatch } from 'redux'
+import { CurrencyRatePayload } from 'src/logic/currencyValues/store/reducer/currencyValues'
+import { AppReduxState } from 'src/store'
 
 const fetchCurrencyRate = (safeAddress: string, selectedCurrency: string) => async (
```

Comment:
It's possible to type the return of the function? I think should be something like 
```
const fetchCurrencyRate = (safeAddress: string, selectedCurrency: string): () => void => async(..
```